templates/zerver/help/collapse-a-message.md

```diff
@@ -0,0 +1,8 @@
+## Collapse a message
+
+Zulip allows you to collapse messages to reduce message clutter.
+
+You can do this by simply hovering your mouse over the message, pressing the down chevron button near the upper right corner
+![down chevron](/static/images/help/down_chevron.png) and selecting **-Collapse**.
```

Comment:
I wound recommend putting the image of the down chevron button right after **down chevron button (image)**, where the image is in parentheses.